File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for cmd in commands:
        first = cmd[0] - 1  
        last = cmd[1]       
        find_idx = cmd[2] - 1
        
        srt = sorted(array[first:last]) 
        answer.append(srt[find_idx])
        
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    
    for (vector<int> command : commands) {
        int start = command[0] -1;
        int end = command[1];
        int idx = command[2] -1;
        
        vector<int> temp(array.begin() + start, array.begin() + end); 
        sort(temp.begin(), temp.end());
        answer.push_back(temp[idx]);
    }
    
    return answer;
}